src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.html

```diff
@@ -73,10 +73,17 @@ <h3 class="panel-title">
             <span class="required"></span>
```

Comment:
I suggest to remove the 'Required' labels for 'Pool' and 'Data pool' in the edit mode.

Reply:
Done